starfish/intensity_table.py

```diff
@@ -338,7 +360,7 @@ def _intensities_from_regions(self, props, reduce_op='max') -> "IntensityTable":
         """turn regions back into intensities by reducing over the labeled area"""
         raise NotImplementedError
 
-    def to_dataframe(self) -> pd.DataFrame:
+    def features_to_dataframe(self) -> pd.DataFrame:
```

Comment:
I like typing something.to, and having it tab-complete, to figure out how I can convert some wacky object that i don't understand into a less wacky object that i may potentially understand better. is there a name here that starts w/ to? e.g., to_dataframe_features, to_features_dataframe, etc.?

Reply:
hah, this is a great point. `to_features_dataframe` it is.